cmd/uplink/cmd/meta_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package cmd_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/internal/testrand"
+)
+
+func min(x, y int) int {
+	if x < y {
+		return x
+	}
+
+	return y
+}
+
+func TestSetGetMeta(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount:   1,
+		StorageNodeCount: 14,
+		UplinkCount:      1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		uplinkExe := ctx.Compile("storj.io/storj/cmd/uplink")
```

Comment:
The `ctx` doesn't exist at the top of the test. I don't see a way to pass one into `Run` either. Why would the callback get called more than once?

Reply:
Because the idea for `testplanet.Run` is to iterate through different configurations. Previously it did `sqlite` and `postgres` backends. Ah... right didn't notice the `ctx`... yeah, we can leave it as is  for now... the package compilations should be cached, but we'll see whether it'll end up slow.